.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -1,28 +1,18 @@
 ---
-name: Feature issue template
-about: A broad template for feature-building issues
+name: Issue Template
+about: Each issue should have at least two labels (one to indicate the corresponding epic and one to indicate the nature of the ticket, e.g. client-side or server-side). GitHub milestones correspond to features of the product and issues will be grouped by the feature they relate to.
```

Comment:
Does this need updating with your latest thinking on milestones?

Reply:
This is the update, no?